P5/Source/Guidelines/en/ND-NamesDates.xml

```diff
@@ -1064,7 +1065,79 @@ exemplifying here too? -->
+          how they wish to be identified, without having to share
+          their gender identity (though some do). Cisgender people
+          have also adopted the practice, which normalizes the idea
+          that a person's personal pronouns should not be inferred by
+          their name, sex, gender, or gender presentation. The
+          <gi>persPronouns</gi> element may be used either in
+          transcribed content to encode a phrase used to indicate
+          preferred personal pronouns, or may be used inside a
+          <gi>person</gi> or <gi>persona</gi> element to indicate
+          the associated pronouns.</p>
+
+          <p>For example, the following entry from a hypothetical
+          prosopography lists only the nominative case of the
+          preferred pronouns as identified by Miss Major
+          Griffin-Gracy, a historical figure.
+	  <!-- Source for the following <egXML>: -->
+	  <!-- <bibl xml:id="issue2010">
+	       <author>Clark, Ash</author>. 
+	       <title>Proposal: new element <gi>persPronouns</gi></title>
+	       <ptr target="https://github.com/TEIC/TEI/issues/2010#issue-647632134"/>.
+	       </bibl> -->
+          <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#NONE">
+            <person>
+              <persName>
+                <forename>Miss Major</forename>
+                <surname>Griffin-Gracy</surname>
+              </persName>
+              <birth when="1940-10-25"/>
+              <sex value="transFemale" evidence="selfIdentification">trans woman</sex>
+              <persPronouns value="she" evidence="selfIdentification"/>
+              <note>
+                <p>Veteran of the Stonewall Riots. Founder of the
+                Griffin-Gracy Educational Retreat and Historical
+                Center (the House of GG). Activist and advocate for
+                transgender and gender-nonconforming people of
+                color.</p>
+              </note>
+            </person>
+          </egXML>
+          </p>
+	  <p>Personal pronouns often occur as part of the closer of an
+	  email, post, or other electronic communication.
+	  <!-- Syd &amp; Meaghan are working on replacing or
+	       augmenting following example with a transcription of
+	       https://www6.unca.edu/ohms/viewer.php?cachefile=APOH001.xml
+	  -->
+	  <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#TEI-L_b092ac32.2101">
+      <div type="email">
+        <opener><salute>Dear all,</salute></opener>
+        <p>With apologies for length. I'm expanding a schema …</p>
+        <!-- ... -->
+        <closer>
+          <lb/><persName>Diane Jakacki, Ph.D.</persName> 
+          <lb/><roleName>Digital Scholarship Coordinator</roleName> 
+          <lb/><roleName>Affiliate Faculty in Comparative &amp; Digital Humanities</roleName>
+          <lb/><orgName>Bucknell University</orgName>
+          <lb/><email>d…@….edu</email>
+          <lb/><persPronouns>(she/her/hers)</persPronouns>
```

Comment:
maybe move the parentheses out of the element, like 
```xml
(<persPronouns>she/her/hers</persPronouns>)
```
?

Reply:
Eh, I would not. But I do not think it matters much: although of the two I prefer the encoding with parens as part of the content, and believe that the _right_ encoding is with the parens expressed on a renditional attribute, having the parens on the outside is perfectly acceptable, too.